examples/record-type-inclusion/record_type_inclusion.description

```diff
@@ -0,0 +1,6 @@
+// Record type inclusion provides a convenient way to copy the fields defined in a record type descriptor
+// to another record type descriptor. It is equivalent to explicitly defining those fields in the new type descriptor.
```

Comment:
```suggestion
// to another record-type descriptor. It is equivalent to explicitly defining those fields in the new type descriptor.
```

Reply:
It's not hyphenated

Reply:
It should be hyphenated since it is an adjective for the noun descriptor.